docs_user/modules/openstack-rolling_back.adoc

```diff
@@ -0,0 +1,177 @@
+[id="rolling-back-control-plane_{context}"]
+
+= Rolling back the control plane adoption
+
+If you encountered a problem during the adoption of the OpenStack
+control plane services that prevents you from following the adoption
+procedure to completion, you can roll back the control plane adoption.
+
+*Important: The roll back operation is only possible during the
+control plane parts of the adoption procedure. Altering the data plane
+nodes in any way during the procedure marks the point of no return and
+the roll back is no longer possible after that.*
+
+== Overview of the rollback procedure
+
+During the control plane adoption, services on the source cloud's
+control plane are stopped but not removed. The databases on the source
+control plane are not edited by the adoption procedure. The
+destination podified control plane received a copy of the original
+control plane databases. The roll back procedure assumes that the data
+plane has not yet been touched by the adoption procedure and it is
+still connected to the source control plane.
+
+The rollback procedure therefore consists of:
+
+* Restoring the functionality of the source control plane.
+
+* Removing the partially or fully deployed destination podified
+  control plane.
+
+== Control plane rollback procedure
+
+=== Restoring the source control plane
+
+To restore the source cloud to a working state, start the OpenStack
+control plane services that you previously stopped during the adoption
+procedure:
+
+----
+ServicesToStart=("tripleo_horizon.service"
+                "tripleo_keystone.service"
+                "tripleo_cinder_api.service"
+                "tripleo_cinder_api_cron.service"
+                "tripleo_cinder_scheduler.service"
+                "tripleo_cinder_backup.service"
+                "tripleo_glance_api.service"
+                "tripleo_manila_api.service"
+                "tripleo_manila_api_cron.service"
+                "tripleo_manila_scheduler.service"
+                "tripleo_neutron_api.service"
+                "tripleo_nova_api.service"
+                "tripleo_placement_api.service"
+                "tripleo_nova_api_cron.service"
+                "tripleo_nova_api.service"
```

Comment:
nova_api.service is duplicated in the list.

Reply:
Thanks, i added a commit to fix service lists into a WIP branch here https://github.com/jistr/data-plane-adoption/tree/f-rollback-tests